webpack.config.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  entry     : 'main.js', resolve : {
+  root : 'src/UI',
+  alias : {
+      'templates'               : '/templates',
+      'backbone'                : 'Shims/backbone',
+      'moment'                  : 'JsLibraries/moment',
+      'filesize'                : 'JsLibraries/filesize',
+      'handlebars'              : 'Shared/Shims/handlebars',
+      'handlebars.helpers'      : 'JsLibraries/handlebars.helpers',
+      'bootstrap'               : 'JsLibraries/bootstrap',
+      'backbone.deepmodel'      : 'JsLibraries/backbone.deep.model',
+      'backbone.pageable'       : 'JsLibraries/backbone.pageable',
+      'backbone-pageable'       : 'JsLibraries/backbone.pageable',
+      'backbone-pageable'       : 'JsLibraries/backbone.pageable',
```

Comment:
so there was a reason for this, depending on modules (3rd party) that depend on backbone.pagable and how they reference it. sometime they require it by `-` sometime by `.`

we can re-evalute this, but this is straight from the require js config.